### PR TITLE
Start Policy Studio Wrapper

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/6370

**Description**

 - Start Policy Studio Wrapper
 - Create required services using Angular

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-albxwidcvm.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-migrate-policy-studio/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-majislmyrb.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-viobmjqpxs.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-migrate-policy-studio/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-lvfuxlcjpe.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-migrate-policy-studio/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-pdliajxfmv.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-migrate-policy-studio/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/components/old-policy-studio-wrapper/old-policy-studio-wrapper.component.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import ResourceService from '../../services/resource.service';
 import SpelService from '../../services/spel.service';
 import '@gravitee/ui-components/wc/gv-policy-studio';
 
-const PolicyStudioWrapperComponent: ng.IComponentOptions = {
+const OldPolicyStudioWrapperComponent: ng.IComponentOptions = {
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/components/old-policy-studio-wrapper/old-policy-studio-wrapper.component.ts:22]: `Ajs` prefix :p  but old is good too

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**